File: check if two strings arrays are equal.py
class Solution(object):
    def arrayStringsAreEqual(self, word1, word2):
        w1 = ""
        w2 = ""
        for word in word1:
            w1 += word
        for word in word2:
            w2 += word
        
        return w1 == w2

s = Solution()
# Example usage:
print(s.arrayStringsAreEqual(["ab", "c"], ["a", "bc"]))
# Example usage:
print(s.arrayStringsAreEqual(["a", "b"], ["ab"]))

File: check if two strings arrays are equal.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        StringBuilder sb1 = new StringBuilder();
        StringBuilder sb2 = new StringBuilder();
        for (int i = 0; i<word1.length; i++){
            sb1.append(word1[i]);
        }
        for (int i = 0; i<word2.length; i++){
            sb2.append(word2[i]);
        }

        return sb1.toString().equals(sb2.toString());
        
    }
}